packages/terra-consumer-nav/src/components/nav-items/NavItems.jsx

```diff
@@ -42,14 +46,36 @@ class NavItems extends Component {
     this.handleToggle = this.handleToggle.bind(this);
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (this.props.currentURL === nextProps.currentURL) {
+      return;
+    }
+
+    const currentComponent = this;
+    const currentURL = nextProps.currentURL;
+
+    /* if there is a sub nav which matches the currentURL open the parent nav */
+    currentComponent.props.navItems.find((element, i) => {
+      if (element.subItems) {
+        element.subItems.find((item) => {
```

Comment:
this is not the right way to use `find`. the callback passed to `find` should return a boolean value. Code should be changed to something like the following:
```
const match = elements.subItems.find(item=> item.url === currentURL);
if (match) {
  currentComponent.setState({openToggle: i});
}
```

Reply:
Also, since `find` method is not natively supported in IE (see [ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find#Browser_compatibility)), you may need add some polyfill for it and should test it in IEs that terra-consumer supports.

Reply:
Updated!  Also, as our application is consuming this package and uses polyfill it supports {code}find{code} method in IE, but if someone wants to consume this polyfill is required.